conformance.adoc

```diff
@@ -37,6 +37,7 @@ See https://github.com/ugrid-conventions/ugrid-conventions for the UGRID conform
 *Recommendations:*
 
 * Variable, dimension and attribute names should begin with a letter and be composed of letters (A-Z, a-z), digits (0-9), and underscores(_). This corresponds to ASCII characters in the decimal ranges (65-90), (97-122), (48-57), and (95). The corresponding Unicode codepoints are (U+0041-U+005A), (U+0061-U+007A), (U+0030-U+0039), and (U+005F).  
+* ASCII period (.) and ASCII hyphen (-) may also be included in attribute names only.
```

Comment:
Add decimal codes and Unicode codepoints as per [suggestion ](https://github.com/cf-convention/cf-conventions/issues/477#issuecomment-1965251024)

Reply:
I respectfully disagree with your requested changes.  The text is clear as I have drafted.